src/openrct2/entity/Guest.cpp

```diff
@@ -2959,10 +2959,17 @@ static PeepThoughtType peep_assess_surroundings(int16_t centre_x, int16_t centre
                         if (tileElement->AsPath()->IsBroken())
                         {
                             num_rubbish++;
+                            break;
+                        }
+                        if (pathAddEntry->flags & (PATH_BIT_FLAG_LAMP))
+                        {
+                            num_scenery++;
                         }
                         break;
                     }
                     case TileElementType::LargeScenery:
+                        num_scenery += 2;
+                        break;
                     case TileElementType::SmallScenery:
                         num_scenery++;
```

Comment:
I feel like guests think this way too quickly. This line suggests trees also count as scenery, maybe these should be excluded? Flowers should still count, in my opinion.

Reply:
Counting or not counting trees is tricky, I think.

A) Yeah, guests probably shouldn't be thinking 'nice scenery' just because they're walking through a forest of randomly spammed pine trees. (Although, I suppose there *is* an argument for this. It would be like a nice hike in the woods, right?)

B) But sometimes trees *can* be an integral part of a really nice scenery setup. Especially the more floral/decorative trees. (And also because some things like a lot of the candy land scenery count as trees.)

Reply:
That's interesting @Broxzier, my feeling was completely opposite (regarding how fast this thought occurs). Also, I really _do_ believe trees should count for the calculation. 

For example, a theme park like Efteling in the Netherlands has a lot of trees. In many areas you could see these as 'randomly spammed', as @ocalhoun6 puts it. But in my experience, many guests really appreciate this as part of the scenery.